Trap Quest.inform/Inform/Extensions/Every Turn/Compute Turn.i7x

```diff
@@ -938,7 +938,8 @@ To Reset Flags:
 		if being-fucked is 0, now the buildup of F is 0;
 	if the rawness of penis > 0 and wanking is 0 and the number of things penetrating penis is 0 and the number of things penetrating vagina is 0:
 		RawDown penis.
-
+	repeat with A running through bsounding clothing:
+		now A is not bsounding;
```

Comment:
This needs to go before line 939, since the block currently ends at line 939 & 940. So these two lines are just floating in the air at the moment.